### ISSUE TITLE
add the rawIdentifier to the XML page source

### ISSUE THREAD

**Issue by toejough** [facebook/WebDriverAgent#420]
I'm digging around the code to see where I might submit a PR, but I'd like to request that the rawIdentifier get added to the XML page source.

I use the page source command quite liberally to get a bunch of screen context all at once, and my dev team has started using the rawIdentifier more frequently, and I don't have access to it through the page source.  At the moment, the 'name' field gets set to the same value in our app, but I don't want to depend on that forever.

**Comment by mykola-mokhnach** [comment]
I don't see any property in XCUIElementAttributes protocol, which allows to read _rawIdentifier_. We can only use the stuff, that Apple's XCTest framework provides to us.

**Comment by marekcirkos** [comment]
Inspector has 'rawIdentifier' identifier that returns unmodified name identifier property. However @toejough I don't understand why you can't use name or identifier? It guarantees WD spec compliance, rawIdentifier is whatever Apple wants to return.

**Comment by marekcirkos** [comment]
@toejough  ping

**Comment by toejough** [comment]
sorry.

![screen shot 2017-01-10 at 2 57 49 pm](https://cloud.githubusercontent.com/assets/5317312/21822454/33f223ca-d745-11e6-921c-a5acdcb944bc.png)

That screenshot is from the inspector for our app.  So far, all the `Raw identifier` values have matched the `name` values I've seen for elements, so I've been using `name` from the xml.  

Partly, this may just be confusion on my part over what these fields are.  My devs have recently started telling me they want to use the "identifier" on their elements instead of the "accessibility label", and asked if I could access the "identifier".  I found `Raw identifier` on some inspector elements, and assumed that was the field they were starting to set, and that while `name` was the same value right now, it might not always be.

After getting your response I started to doubt that the names lined up like I thought and asked my devs for docs for the fields they were setting in the app, and they referred me to this:

https://developer.apple.com/reference/uikit/uiaccessibilityidentification/1623132-accessibilityidentifier

What does WDA call that field?  If that's just the `name` field I've been using the whole time, then we can just close this issue.
